### PR TITLE
refactor(demultiplexing cli commands)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

Refactoring of demultiplexing cli commands:

`cg demultiplex flow-cell <flow-cell-name>` -> `cg demultiplex sequencing-run <sequencing-run-name>`  

`cg demultiplex all <flow-cells-dir>` -> `cg demultiplex all <sequencing-runs-dir>`

`cg demultiplex copy-completed-flow-cells` -> `cg demultiplex copy-completed-sequencing-runs`

`cg demultiplex confirm-flow-cell-sync`-> `cg demultiplex confirm-sequencing-run-sync`

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/cli/demultiplex/demux.py [hunk 1]
```diff
@@ -75,26 +75,28 @@ def demultiplex_all(context: CGConfig, flow_cells_directory: click.Path, dry_run
             )
 
 
-@click.command(name="flow-cell")
-@click.argument("flow-cell-name")
+@click.command(name="sequencing-run")
+@click.argument("sequencing-run-name")
 @DRY_RUN
 @click.pass_obj
 def demultiplex_flow_cell(
```
Comment (diitaz93) [inline cg/cli/demultiplex/demux.py:82]: supposed to be changed too?

File: cg/cli/demultiplex/demux.py [hunk 2]
```diff
@@ -75,26 +75,28 @@ def demultiplex_all(context: CGConfig, flow_cells_directory: click.Path, dry_run
...
 @DRY_RUN
 @click.pass_obj
 def demultiplex_flow_cell(
     context: CGConfig,
     dry_run: bool,
-    flow_cell_name: str,
+    sequencing_run_name: str,
 ):
-    """Demultiplex a flow cell using BCLConvert.
+    """Demultiplex a sequencing using BCLConvert.
```
Comment (diitaz93) [inline cg/cli/demultiplex/demux.py:87]:
```suggestion
    """Demultiplex an Illumina sequencing run using BCLConvert.

```

**Review by diitaz93 — APPROVED** [review]
Looks good 👍

**Comment by ChrOertlin** [comment]
Goes with: https://github.com/Clinical-Genomics/servers/pull/1382

**Review by Vince-janv — APPROVED** [review]
👍

**Comment by ChrOertlin** [comment]
# tests

image

**Pull request merged**